client/client_test.go

```diff
@@ -202,6 +203,33 @@ func Test_Client_Marshal(t *testing.T) {
 		require.Equal(t, errors.New("empty xml"), err)
 	})
 
```

Comment:
This is missing a test using the default CBOR Marshall and UnMarshall functions as noted by `codecov` below.